network/utils.go

```diff
@@ -56,18 +56,6 @@ func customSchemeToBaseGolangScheme(scheme string) string {
 	return scheme
 }
 
-// ClientIDConnectionWrapper callback that wraps connections with clientID, to
-// allow extracting it later. Is used in the contexts without TLS, because the
-// latter has its own mechanism of extracting the clientID.
-type ClientIDConnectionWrapper struct {
```

Comment:
No, it doesn't, because I've added this struct in the PR earlier - https://github.com/cossacklabs/acra/pull/550/commits/bb2f5dcd79bdf210cffe12022e17fc3ffc32e0ce
But even if it broke something, I believe we would catch it by tests or with compilation error.